ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_tensor.py

```diff
@@ -1285,3 +1288,45 @@ def test_paddle_bitwise_xor(
         frontend=frontend,
         on_device=on_device,
     )
+
+
+# rot90
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="paddle.to_tensor",
+    method_name="rot90",
+    dtype_m_k_axes=_get_dtype_values_k_axes_for_rot90(
+        available_dtypes=helpers.get_dtypes("numeric"),
```

Comment:
We should be ideally testing for all valid dtypes, and not only a subset.

Reply:
The numeric has included  float, integer, bool types that are all supported by Rot90.  you can check https://docs.python.org/3/library/stdtypes.html#numeric-types-int-float-complex. Please free free to ask me any questions. Thank you!

Reply:
@DragosStoican

Reply:
Thank you very much for your guidance. @DragosStoican  Sorry, I made a mistake. I have fixed it. Could you help me review it again. Thanks!